Access Local tests are broken

```
Chrome 53.0.2785 (Mac OS X 10.11.6) AccessService pingback should wait for first authorization completion FAILED
	AssertionError: expected 0 to equal 1
	    at /var/folders/7w/bvjck7ls7f58m040x_qvph4w002y5_/T/ae7236cba604848a6d4dafe8d91cc60d.browserify:16176:56 <- /Users/mkhatib/amphtml/extensions/amp-access/0.1/test/test-amp-access.js:1041:55
Chrome 53.0.2785 (Mac OS X 10.11.6) AccessService pingback should wait for last authorization completion FAILED
	AssertionError: expected 0 to equal 1
	    at /var/folders/7w/bvjck7ls7f58m040x_qvph4w002y5_/T/ae7236cba604848a6d4dafe8d91cc60d.browserify:16201:56 <- /Users/mkhatib/amphtml/extensions/amp-access/0.1/test/test-amp-access.js:1067:55
Chrome 53.0.2785 (Mac OS X 10.11.6) WebLoginDialog should succeed with URL promise FAILED
	TypeError: windowApi.messageListener is not a function
	    at succeed (/var/folders/7w/bvjck7ls7f58m040x_qvph4w002y5_/T/ae7236cba604848a6d4dafe8d91cc60d.browserify:17492:15 <- /Users/mkhatib/amphtml/extensions/amp-access/0.1/test/test-login-dialog.js:207:14)
	    at /var/folders/7w/bvjck7ls7f58m040x_qvph4w002y5_/T/ae7236cba604848a6d4dafe8d91cc60d.browserify:17650:7 <- /Users/mkhatib/amphtml/extensions/amp-access/0.1/test/test-login-dialog.js:403:10
```

--- a/extensions/amp-access/0.1/test/test-amp-access.js
+++ b/extensions/amp-access/0.1/test/test-amp-access.js
@@ -907,7 +907,7 @@ describe('AccessService pingback', () => {
 
     const adapter = {
       isPingbackEnabled: () => true,
-      pingback: () => {},
+      pingback: () => Promise.resolve(),
     };
     service.adapter_ = adapter;
     adapterMock = sandbox.mock(adapter);
@@ -1036,7 +1036,8 @@ describe('AccessService pingback', () => {
       expect(service.reportViewToServer_.callCount).to.equal(0);
       expect(triggerEventStub.callCount).to.equal(triggerStart);
       firstAuthorizationResolver();
-      return service.firstAuthorizationPromise_;
+      return Promise.all([service.reportViewPromise_,
+          service.reportViewPromise_]);
     }).then(() => {
       expect(service.reportViewToServer_.callCount).to.equal(1);
       expect(triggerEventStub.callCount).to.equal(triggerStart + 1);
@@ -1062,7 +1063,8 @@ describe('AccessService pingback', () => {
       expect(service.reportViewToServer_.callCount).to.equal(0);
       expect(triggerEventStub.callCount).to.equal(triggerStart);
       lastAuthorizationResolver();
-      return service.lastAuthorizationPromise_;
+      return Promise.all([service.lastAuthorizationPromise_,
+          service.reportViewPromise_]);
     }).then(() => {
       expect(service.reportViewToServer_.callCount).to.equal(1);
       expect(triggerEventStub.callCount).to.equal(triggerStart + 1);

--- a/extensions/amp-access/0.1/test/test-amp-access.js
+++ b/extensions/amp-access/0.1/test/test-amp-access.js
@@ -1036,7 +1036,7 @@ describe('AccessService pingback', () => {
       expect(service.reportViewToServer_.callCount).to.equal(0);
       expect(triggerEventStub.callCount).to.equal(triggerStart);
       firstAuthorizationResolver();
-      return Promise.all([service.reportViewPromise_,
+      return Promise.all([service.firstAuthorizationPromise_,
           service.reportViewPromise_]);
     }).then(() => {
       expect(service.reportViewToServer_.callCount).to.equal(1);

